app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettingsFragment.kt

```diff
@@ -128,8 +132,7 @@ class WallpaperSettingsFragment : Fragment() {
                             val retryResult = wallpaperUseCases.selectWallpaper(wallpaper)
                             onWallpaperSelected(wallpaper, retryResult, view)
                         }
-                    }
-                    .show()
```

Comment:
Let's avoid this change since we didn't have to touch this code. I am curious if this was automatically reformatted via your Android Studio since we recently ran a mass ktlint formatting so this seems a bit unexpected to me.

Reply:
it was semi-intentional, reverted